google-cloud-bigtable/lib/google/cloud/bigtable/service.rb

```diff
@@ -68,7 +68,12 @@ def initialize project_id, credentials, host: nil, timeout: nil,
 
         def channel default_host
           require "grpc"
-          GRPC::Core::Channel.new((host || default_host), nil, chan_creds)
+          GRPC::Core::Channel.new((host || default_host), chan_args, chan_creds)
+        end
+
+        def chan_args
+          { "grpc.max_send_message_length"    => -1,
```

Comment:
Is this something that we should do on all veneer grpc clients? Do we ever want to keep the grpc’s 4MB limit?

Reply:
New question issue?

Reply:
I just ran into the same situation for `google-cloud-text_to_speech` client, should I open a question issue and/or a PR?

Reply:
@benoror Please open a question issue. That is a generated package so a PR with direct manual changes to the generated code might not be the right solution. Thanks!!

Reply:
FWIW in the case of TTS it's possible to pass a GPRC::Channel with overriden args: https://github.com/googleapis/google-cloud-ruby/issues/2408#issuecomment-418858124